elements/lisk-p2p/test/integration/custom_peer_selection.ts

```diff
@@ -25,11 +25,14 @@ import {
 	P2PPeerSelectionForConnectionInput,
 } from '../../src/p2p_types';
 
+import {
```

Comment:
Why not using `createNetwork` & `destroyNetwork` here?

Reply:
Oh, I somehow I missed to add the changes while committing, will do